tests/vector_stores/test_google.py

```diff
@@ -26,14 +28,22 @@
     import llama_index.vector_stores.google.generativeai.genai_extension as genaix
 
     # Make sure the tests do not hit actual production servers.
-    genaix.set_defaults(
+    genaix.set_config(
```

Comment:
Nit: Maybe use set_google_config and GoogleConfig instead?

Reply:
Done.

Reply:
undid this because of the imports issue :)